index.js

```diff
@@ -1,21 +1,86 @@
-document.getElementById('open-bar').onclick = function() {
-    document.getElementById('menu-bar').classList.add('show')
-    document.getElementById('open-bar').classList.add('close');
-    document.getElementById('close-bar').classList.remove('close');
-    // document.getElementById('navi').classList.add('navi');
-    
-}
-
-document.getElementById('close-bar').onclick = function(){
-    document.getElementById('menu-bar').classList.remove('show')
-    document.getElementById('open-bar').classList.remove('close');
-    document.getElementById('close-bar').classList.add('close');
-    // document.getElementById('navi').classList.remove('navi');
-}
-
-document.getElementById('toggle').onclick=function(){
+/* eslint-disable prefer-destructuring */
+document.getElementById('open-bar').onclick = () => {
+  document.getElementById('menu-bar').classList.add('show');
+  document.getElementById('open-bar').classList.add('close');
+  document.getElementById('close-bar').classList.remove('close');
+};
+
+document.getElementById('close-bar').onclick = () => {
+  document.getElementById('menu-bar').classList.remove('show');
+  document.getElementById('open-bar').classList.remove('close');
+  document.getElementById('close-bar').classList.add('close');
+};
+
+document.getElementById('toggle').onclick = () => {
+  document.getElementById('menu-bar').classList.remove('show');
+  document.getElementById('open-bar').classList.remove('close');
+  document.getElementById('close-bar').classList.add('close');
+};
+
+document.getElementById('toggle1').onclick = () => {
```

Comment:
- [ ] I cannot open this project's modal menu kindly make sure they are also accessible. Kindly watch the video:

![mobile (1)](https://user-images.githubusercontent.com/69011963/210369786-4197187b-4d1f-406e-bfd2-9b6c3d75e372.gif)